src/routes/subRecipe.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const subRecipe = require("../usecases/recipe");
```

Comment:
Estas segura que debes llamar los casos de uso de recipe? o de subRecipe?
```suggestion
const subRecipe = require("../usecases/subRecipes");
```